components/playlist/browser/resources/components/app.v1.tsx

```diff
@@ -0,0 +1,81 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { bindActionCreators, Dispatch } from 'redux'
+import { connect } from 'react-redux'
+import { Route, Switch } from 'react-router-dom'
+import styled from 'styled-components'
+
+// Types
+import { Playlist } from '../../../../definitions/playlist'
+import * as playlistActions from '../actions/playlist_action_creators'
+
+// Components
+import Header from './header'
+import PlaylistsCatalog from './playlistsCatalog'
+import VideoFrame from './videoFrame'
+import PlaylistItem from './playlistItem'
+
+interface Props {
+  actions: any
+  playlistData: Playlist.State
+}
+
+const HeaderWrapper = styled.header`
+  position: fixed;
+  width: 100%;
+  height: 56px;
+  top: 0;
+  z-index: 1;
+`
+
+const SectionWrapper = styled.section`
+  margin-top: 56px;
+`
```

Comment:
I'm not confident what would be elegant way to make fixed header style. I'm all ears, so please let me know if I can do better